src/index.ts

```diff
@@ -33,14 +39,16 @@ export const run = () => {
   }
 };
 
-const showHelp = (cmdPassed: string, helpFlag: boolean) => {
-  if (cmdPassed) {
-    console.log(colors.red(`unknown command: ${cmdPassed}`), '\n');
-  } else if (!helpFlag) {
+const showHelp = (args: string[], helpFlag: boolean) => {
+  if (args.length > 2) {
+    console.log(colors.red('Too many arguments passed.'), '\n');
+  } else if (args[0]) {
+    console.log(colors.red(`unknown command: ${args[0]}`), '\n');
+  } else if (!args[0] && !helpFlag) {
```

Comment:
The logic here is getting confusing and requires the context of the code calling this function to fully understand what's going on in this. By just looking at this function you would not understand why it is printing an error if `arg[0]` is present, for example, which is not ideal.

Understanding a function's logic should not require us to know the context of the code from where it is called. So, we should probably move these error messages to the `if-else` block itself in the `run` function. Also, split the `|| args.length > 2` clause into a separate `else if` block for better readability.

Reply:
Also, instead of just checking `args.length > 2`, we should probably also check `args[0] === 'ios' && args.length > 1`, so the new `else if` statement would be:

```ts
} else if (args.length > 2 || (args[0] === 'ios' && args.length > 1)) {
```